### ISSUE TITLE
This way, the DOM object of the background page is obtained using jquery!

### ISSUE THREAD

**Issue by phpjishu** [nwjs/nw.js#6120]
This way, the DOM object of the background page is obtained using jquery!
`<script>
			var $ = jQuery = require('jquery');
			console.log($('body'));
		</script>`

**Comment by bambooCZ** [comment]
Yep, when you use require, tken everything required is evaled in background window context. Try to set `window.new_instance: true` in your `package.json`, or try to play with `mixed_content` or learn to live with that... or wait for NWjs officials to repair it.

**Comment by Christywl** [comment]
Try the above codes, and I can see the console log in the dev tools. @rogerwang , could you please confirm this?

**Comment by bambooCZ** [comment]
I can confirm this. This is consistent behavior since v0.13. Can I have a little requirement? Can you fix this, but allow me to get back old (I mean current) behavior? I mean any kind of flag, chromium-arg, etc? I am afraid this would break our application. Since we have a "legacy" version for Windows < 7 built on top of NWjs v0.14.3 it would be a nightmare to maintain it.
